workers/datasets_based/src/datasets_based/worker.py

```diff
@@ -296,6 +311,13 @@ def process(
             try:
                 self.pre_compute()
                 content = self.compute()
+
+                # Validate content size
```

Comment:
Maybe move this to the libcommon.cache -> it's more a limitation of the cache backend than of the worker as itself.

Reply:
I moved the validation to simple cache, I will need to pass the max configuration since `CachedResponse`  inherits from Document and couldn't have any env configuration inside at least I wouldn't like to mix db engine with config logic values

Reply:
As per last commit, I kept  the logic in the worker

Reply:
Yes, thanks